system-parachains/coretime/coretime-polkadot/src/coretime.rs

```diff
@@ -335,5 +347,9 @@ impl pallet_broker::Config for Runtime {
 	type WeightInfo = weights::pallet_broker::WeightInfo<Runtime>;
 	type PalletId = BrokerPalletId;
 	type AdminOrigin = EnsureRoot<AccountId>;
+	type SovereignAccountOf = SovereignAccountOf;
+	// Roughly 1.5 years with a period of 28 days(~ish)
```

Comment:
The `MaxAutoRenewals` refers to the number of cores for which auto-renewal can be enabled.
```suggestion
```